config/routes.rb

```diff
@@ -2,7 +2,7 @@
 
   resources :deck_favorites, only: [:create, :destroy]
 
-  root 'dashboard#show'
+  root 'pages#landing'
```

Comment:
I need to read into it more. For now I prefer single quotes for non interpolated strings